angular-workspace/projects/ni/nimble-angular/pipes/duration.pipe.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DurationPipe } from './duration.pipe';
+
+describe('DurationPipe', () => {
+    describe('basic formatting (English)', () => {
+        let pipe: DurationPipe;
+
+        beforeEach(() => {
+            pipe = new DurationPipe('en');
```

Comment:
Could be reused by all tests, since it should not have any state.

Reply:
Updated